index.js

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import { render } from 'react-dom';
 
 // START: test components to try rendering:
-// import TestMDX from './examples/mdx/test-mdx';
-import TestJs from './examples/js/test-js';
+import TestMDX from './examples/mdx/test-mdx';
+// import TestJs from './examples/js/test-js';
```

Comment:
At some point, we should add an index page that lets you pick which test you want to render.